src/lib/y2security/autoinst_profile/security_section.rb

```diff
@@ -18,53 +18,31 @@
 # find current contact information at www.suse.com.
 
 require "installation/autoinst_profile/section_with_attributes"
-require "y2security/autoinst_profile/lsm_section"
 
 module Y2Security
   module AutoinstProfile
     # This class represents an AutoYaST <security> section although by now it only handles
     # LSM related attributes
     #
     # <security>
-    #   <!-- <selinux_mode></selinux_mode> # Deprecated -->
-    #   <lsm>
-    #     <apparmor>
-    #       <selectable config:type="boolean">false</selectable>
-    #     </apparmor>
-    #     <selinux>
-    #       <mode>permissive</mode>
-    #       <configurable config:type="boolean">true</configurable>
-    #       <patterns>selinux</patterns>
-    #     </selinux>
-    #   </lsm>
+    #   <selinux_mode>enforcing</selinux_mode>
+    #   <lsm_select>apparmor</lsm_select>
```

Comment:
does example makes sense at all? with apparmor I expect that selinux mode is not selected at all and maybe show warning?

Reply:
No you are right. The example does not make sense and I added for showing both options but should use **SELinux** in this case.

We were discussing about the different options. The user can define both as the example shows, and that would influence the default **SELinux** mode exposed in the proposal but AppArmor will be selected, so, only if the selection is modified the value will be relevant and @dgdavid preferred to just ignore the option if the **LSM** selected explicitly is different to **SELinux**

I guess that we could see it as the **lsm** kernel option which has precedence over **security** parameter and which invalidates any **SELinux** option if it is not the **major** module selected.